master/reference/typha/configuration.md

```diff
@@ -0,0 +1,50 @@
+| `HealthEnabled`                   | `TYPHA_HEALTHENABLED`                   | When enabled, exposes typha health information via an http endpoint. | boolean |
+| `LogFilePath`                     | `TYPHA_LOGFILEPATH`                     | The full path to the Typha log. Set to `none` to disable file logging. [Default: `/var/log/calico/typha.log`] | string |
+| `LogSeverityFile`                 | `TYPHA_LOGSEVERITYFILE`                 | The log severity above which logs are sent to the log file. [Default: `Info`] | `Debug`, `Info`, `Warning`, `Error`, `Fatal` |
+| `LogSeverityScreen`               | `TYPHA_LOGSEVERITYSCREEN`               | The log severity above which logs are sent to the stdout. [Default: `Info`] | `Debug`, `Info`, `Warning`, `Error`, `Fatal` |
+| `LogSeveritySys`                  | `TYPHA_LOGSEVERITYSYS`                  | The log severity above which logs are sent to the syslog. Set to `""` for no logging to syslog. [Default: `Info`] | `Debug`, `Info`, `Warning`, `Error`, `Fatal` |
+| `PrometheusGoMetricsEnabled`      | `TYPHA_PROMETHEUSGOMETRICSENABLED`      | Set to `false` to disable Go runtime metrics collection, which the Prometheus client does by default. This reduces the number of metrics reported, reducing Prometheus load. [Default: `true`]  | boolean |
+| `PrometheusMetricsEnabled`        | `TYPHA_PROMETHEUSMETRICSENABLED`        | Set to `true` to enable the experimental Prometheus metrics server in Typha. [Default: `false`] | boolean |
+| `PrometheusMetricsPort`           | `TYPHA_PROMETHEUSMETRICSPORT`           | Experimental: TCP port that the Prometheus metrics server should bind to. [Default: `9091`] | int |
+| `PrometheusProcessMetricsEnabled` | `TYPHA_PROMETHEUSPROCESSMETRICSENABLED` | Set to `false` to disable process metrics collection, which the Prometheus client does by default. This reduces the number of metrics reported, reducing Prometheus load. [Default: `true`] | boolean |
+
+#### etcdv3 datastore configuration
+
+| Configuration parameter | Environment variable  | Description | Schema |
+| ----------------------- | --------------------- | ----------- | ------ |
+| `EtcdCaFile`            | `TYPHA_ETCDCAFILE`    | The full path to the etcd Certificate Authority certificate file. To disable authentication of the server by Typha, set the value to `none`. [Default: `/etc/ssl/certs/ca-certificates.crt`] | string |
+| `EtcdCertFile`          | `TYPHA_ETCDCERTFILE`  | The full path to the etcd certificate file. | string |
+| `EtcdEndpoints`         | `TYPHA_ETCDENDPOINTS` | Comma-delimited list of etcd endpoints to connect to. Example: `http://etcd1:2379,http://etcd2:2379`. | `<scheme>://<ip-or-fqdn>:<port>` |
+| `EtcdKeyFile`           | `TYPHA_ETCDKEYFILE`   | The full path to the etcd private key file. | string |
+
+#### Kubernetes datastore configuration
```

Comment:
We are trying to standardize this usage to be consistently: "Kubernetes API datastore"

Reply:
Cool, and I'll make that change in the Felix config page too.